va/va.go

```diff
@@ -499,8 +499,8 @@ func (va *ValidationAuthorityImpl) fetchHTTP(ctx context.Context, identifier cor
 	// io.LimitedReader will silently truncate a Reader so if the
 	// resulting payload is the same size as maxResponseSize fail
 	if len(body) >= maxResponseSize {
-		return nil, validationRecords, probs.Unauthorized("Invalid response from %s: q", url,
-			replaceInvalidUTF8(body))
+		return nil, validationRecords, probs.Unauthorized(fmt.Sprintf("Invalid response from %s: %q", url,
```

Comment:
The q/%q thing looks like a real bug; let's give that its own PR. Also, it seems like we don't need the `fmt.Sprintf`, since Unauthorized handles expansions.

Reply:
https://github.com/letsencrypt/boulder/pull/3870